cpp/src/arrow/compute/kernels/scalar_temporal.cc

```diff
@@ -836,6 +841,10 @@ const FunctionDoc strftime_doc{
     "Format timestamps according to a format string",
     ("For each input timestamp, emit a formatted string.\n"
      "The time format string and locale can be set using StrftimeOptions.\n"
+     "Output precision of %S (seconds) flag depends on the input timestamp precision. "
```

Comment:
I don't know if the elaboration about the "%S" is useful to put here, but if it is, can you follow the convention for line wrapping?

Reply:
Updated. Could you please check if it's ok now. I'm not sure about the convention.

Reply:
This evaluates to true in MySQL on sqlfiddle:
```SQL
SELECT WEEK('2008-1-6', 1) != WEEK('2008-1-6', 4)
```

Reply:
> I don't know if the elaboration about the "%S" is useful to put here

@pitrou since this behaves differently as eg Python and R's version of `strftime`, I think it's important to note this somewhere (but can also be somewhere else, eg in compute.rst as a note below the table. EDIT: I see it's already added there)